scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja

```diff
@@ -630,32 +641,25 @@ psa_status_t psa_driver_wrapper_import_key(
             /* Key is stored in the slot in export representation, so
              * cycle through all known transparent accelerators */
 #if defined(PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT)
-#if defined(PSA_CRYPTO_DRIVER_TEST)
-            status = mbedtls_test_transparent_import_key(
-                         attributes,
-                         data, data_length,
-                         key_buffer, key_buffer_size,
-                         key_buffer_length, bits );
-            /* Declared with fallback == true */
-            if( status != PSA_ERROR_NOT_SUPPORTED )
-                return( status );
-#endif /* PSA_CRYPTO_DRIVER_TEST */
+/* BEGIN-Templating */
+{% with nest_indent=12 %}
+{% include "OS-template-transparent.jinja" -%}
+{% endwith -%}
+/* END-Templating */
```

Comment:
The purpose of the comments here are to help as warning aids for driver developers manually updating or editing this file, to tread cautiously. A macro might not serve that purpose.

Reply:
I don't understand why this part of the template needs special attention. And why would driver developers edit the template?

If this is about developers who edit the generated file, whether this part is generated manually or via a template macro doesn't change anything.

Reply:
It's not about developers who edit the generated file... because that just wouldn't fly, it would be easily overwritten. 
But while we continue to work on improving the codegen, developers/ integrators could either replace or edit to add into the main template file, and that would work fine.

The template file could as good as be patched to include hardcoded templating and keeping these comments helps to warn.

Reply:
I still don't understand why this shouldn't be a macro.

> The purpose of the comments here are to help as warning aids for driver developers manually updating or editing this file, to tread cautiously. A macro might not serve that purpose.

There are no Jinja comments here, only C comments. The C comments are part of what is generated, they're not even targeted at people looking at the template, only at people looking at the generated C code. So they're not warnings towards people editing the template.

And I'm not sure why people who edit the template should tread carefully _here_ specifically, anyway.

Reply:
I knocked off the comments. Now it is a tempate inclusion with nest. Am not able to make it a macro, maybe since that in turn includes another file ?
+{% macro one_shot_opaque_dispatch() -%}
+    {% with nest_indent=8 %}
+    {% include "OS-template-opaque.jinja" -%}
+    {% endwith -%}
+{% endmacro %}
Didn't work for me. Is there a better way to do what you're asking for ?